servicecontrol/servicecontrol-instances/hardware.md

```diff
@@ -46,19 +46,19 @@ The embedded RavenDB will use additional RAM to improve indexing performance. Du
 
 ### Message size / MaxBodySizeToStore
 
-In general, the smaller the messages, the faster ServiceControl will be able to process audit records. Thus, [put events on a diet](https://particular.net/blog/putting-your-events-on-a-diet) if optimizing for performance. For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
+In general, the smaller the messages, the faster ServiceControl will process audit records (see [_Putting your events on a diet_](https://particular.net/blog/putting-your-events-on-a-diet)). For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
```

Comment:
@ramonsmits CI fails because of the word "your" in the blog post title. Maybe this instead?

```suggestion
In general, [the smaller the messages](https://particular.net/blog/putting-your-events-on-a-diet), the faster ServiceControl will process audit records. For larger message payloads, consider using the [data bus feature](/nservicebus/messaging/databus/).
```